src/modules/displayLikes.js

```diff
@@ -0,0 +1,14 @@
+// import renderHomePage from "./renderHomePage";
+
+const displayLike = async (likes) => {
+  likes.forEach((like) => {
+    const elementid = `show-${like.item_id}`;
+    const showLike = document.getElementById(elementid);
+    if (showLike !== null) {
+      const likeCount = showLike.querySelector('.like-count');
+      likeCount.innerHTML = `(${like.likes})`;
+    }
+  });
+};
+
+export default displayLike;
```

Comment:
Welldone @Rachelwebdev on building this functional component to update the number of likes when a user clicks on the like button 🙂👍👏. 
However, I noticed that when the user clicks on the button, it takes a long time for the count to be updated. 
This would not be a very good user experience and I think it would be better if you can fix this.

Reply:
Thanks for the review, I would implement these changes